dell/xps/13-9380/default.nix

```diff
@@ -2,6 +2,7 @@
 
 {
   imports = [
+    ../../../common/cpu/intel
```

Comment:
Since all of them also have NVME, I wonder if we should have a NVME profile?

Reply:
You joined the 9380 fan-base? :grinning: 

Yea I think that's fair to say.

Reply:
I was not aware that this is a cult already. The screen is nice. The battery in my old laptop lasted a longer.

Reply:
The NVME profile might be worth another PR.